app/pages/sections/service-recipients/controller.js

```diff
@@ -4,12 +4,34 @@ import { getEndpoint } from '../../../endpoints';
 import { logger } from '../../../logger';
 
 export const getServiceRecipientsContext = async ({ orderId, orgId, accessToken }) => {
-  const serviceRecipientsEndpoint = getEndpoint({ endpointLocator: 'getServiceRecipientsFromOapi', options: { orgId } });
-  const serviceRecipientsData = await getData({
-    endpoint: serviceRecipientsEndpoint,
-    accessToken,
-    logger,
-  });
-  logger.info(`Service recipients for organisation with id: ${orgId} found in OAPI. ${serviceRecipientsData}`);
+  let serviceRecipientsData;
+  let selectedServiceRecipientsData;
+
+  try {
+    const serviceRecipientEndpoint = getEndpoint({ endpointLocator: 'getServiceRecipientsFromOapi', options: { orgId } });
+    serviceRecipientsData = await getData({
+      endpoint: serviceRecipientEndpoint,
+      accessToken,
+      logger,
+    });
+    logger.info(`Service recipients for organisation with id: ${orgId} found in OAPI. ${serviceRecipientsData}`);
+  } catch (err) {
+    logger.error(`No selected service recipients data returned from OAPI for order id: ${orgId}. ${err}`);
+    throw new Error();
+  }
+
+  try {
```

Comment:
shouldn't this try be inside the one above? we don't want to do this if the other one fails

Reply:
true

Reply:
I've removed the first try catch then - it was confusing at the time!